### ISSUE TITLE
Detect author from module fields

### ISSUE THREAD

**Issue by techtonik** [takluyver/flit#118]
Flit could detect Author if it is set as __author__ in module fields.
```
$ flit init
Module name [appvey]:
Author:
```

**Comment by Carreau** [comment]
It already fill in author if you used it more than once, same for email and home page:
```
$ flit init
Module name [dummy]:
Author [Matthias Bussonnier]:
Author email [mysecretmail@thesearchcompany.tld]:
Home page [https://github.com/Carreau/dummy]:
```

**Comment by techtonik** [comment]
```
def get_field(path, name='__version__'):
  '''Read named string from module without importing it'''
  for line in open(path, 'rb'):
    # Decode to unicode for PY2/PY3 in a fail-safe way
    line = line.decode('cp437')
    if line.startswith(name):
      # __version__ = "0.9"
      delim = '\"' if '\"' in line else '\''
      return line.split(delim)[1]
```

**Comment by Carreau** [comment]
> def get_field(path, name='__version__'):

No, we already have a `get_info_from_module` that eval the module and you just need to getattr('the field you are interested in').

> # Decode to unicode for PY2/PY3 in a fail-safe way

No, flit is Python 3 only.

**Comment by takluyver** [comment]
It's not an unreasonable idea, but as @Carreau has heard me saying before: I don't want to put too much smarts into `flit init`. There are a lot of clever things that it could do, and fancier UIs it could use. But I see it as a sideline to the main focus of flit, and more code equals more bugs.

I'd love to see a more general tool which could 'add' various features (flit packaging, license, sphinx docs...) to a project. It could have a fancier UI and do more clever guesswork. There's a great opportunity for someone to build something, I think, but I don't have the time to do it myself.

**Comment by techtonik** [comment]
>  ..that eval the module..

I can't explain, and that's a bad thing, but I consider this a bad design, because target platform for a source Python package may differ, so there is no guarantee that eval should work.

Anyway, I've done what I needed in https://github.com/techtonik/pypack which is unlicensed, so feel free to copy paste from there if you find anything useful.

    git clone https://github.com/techtonik/pypack
    cd pypack
    pypack.py pypack.py
